sos/report/plugins/openstack_mistral.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (C) 2022 Red Hat, Inc.
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin
+import os
+
+
+class OpenStackMistral(Plugin):
```

Comment:
A class docstring would be helpful to detail the plugin a bit more. For example, what is Mistral in the context of both an OSP component and the data collected in an sos report? Take a look at the [kernel plugin](https://github.com/sosreport/sos/blob/main/sos/report/plugins/kernel.py) for a good example of what we look for in new plugin docstrings - these will be used for `sos help` output (e.g. `sos help report.plugins.openstack_mistral`)

Reply:
Right - added one.